lib/govuk_ab_testing/ab_test.rb

```diff
@@ -0,0 +1,23 @@
+module GovukAbTesting
+  class AbTest
+    attr_reader :experiment_name
+
+    def initialize(experiment_name)
+      @experiment_name = experiment_name
+    end
+
+    # Internal name of the header
+    def request_header
+      "HTTP_GOVUK_ABTEST_#{experiment_name.upcase}"
+    end
+
+    def response_header
+      "GOVUK-ABTest-#{cookie_name}"
+    end
+
+    # `example` -> `Example`
+    def cookie_name
+      experiment_name.capitalize
```

Comment:
I have a few minor concerns about this:

- Cookie names are case-sensitive, so this has to exactly match the name in the CDN config. When someone adds a new A/B test, they need to know that this capitalization happens, and that they have to name the experiment `someabtest` and the cookie `ABTest-Someabtest`.
- This method name is misleading because it doesn't return the full cookie name, which includes the `ABTest-` prefix.
- The education cookie will be `ABTest-Educationnavigation` rather than `ABTest-EducationNavigation`. This means it will look a bit clunky in the Chrome extension, where it will be shown as as "Educationnavigation".

I wonder if it would be better to remove `.capitalize`, and tell people that the experiment name in the code must match the cookie name exactly (including case), which is simpler and hopefully less error-prone than if it matched exactly but with the first letter capitalized. And you wouldn't have to worry about this method's name because it won't be necessary any more.